### ISSUE TITLE
Require HTTPS for storage accounts and Function Apps in the deployment templates

### ISSUE THREAD

**Issue by sobjornstad** [tfsaggregator/aggregator-cli#225]
# Feature Request

**Is your feature request related to a problem? Please describe.**

Azure Advisor recommends requiring *HTTPS Only* on function apps and *Secure transfer* on storage accounts for improved security. TFS Aggregator appears to function normally if I turn these options on, but I have to manually reset them if I need to redeploy the Aggregator instance.

**Describe the solution you would like**

Aggregator should set these settings in its default templates, since there are security benefits and no apparent downside except having to set the appropriate flag.

**Describe alternatives you have considered**

One could write a custom script to make the changes after the deploy, but that seems silly.

**Additional context**

Find the settings in the Azure portal here:

![image](https://user-images.githubusercontent.com/2313414/101405658-0c764d80-389e-11eb-9ea0-d19d697ef684.png)

![image](https://user-images.githubusercontent.com/2313414/101405685-14ce8880-389e-11eb-9e1e-764effd28d79.png)

**Comment by giuliov** [comment]
This is available in [v1.0.1](https://github.com/tfsaggregator/aggregator-cli/releases/tag/v1.0.1); would you mind checking it?

**Comment by sobjornstad** [comment]
@giuliov: I'm on holiday break right now, but I'll give it a shot early next year. Thanks!

**Comment by sobjornstad** [comment]
Looks like it worked, thanks!

I am seeing the following odd message on running any command, however:

```
A new version (v1.0.1) of Aggregator CLI is available, please upgrade.
aggregator-cli v1.0.1 (build: 1.0.1.0 Release) (c) Copyright © TFS Aggregator Team
```

As you can see, the CLI identifies itself as 1.0.1, but still says I should upgrade to 1.0.1.

**Comment by giuliov** [comment]
Both issues should be fixed in [1.1](https://github.com/tfsaggregator/aggregator-cli/releases/tag/v1.1.0), can you confirm?
